bindgen/src/bindings/cpp/templates/wrapper.cpp

```diff
@@ -34,6 +57,8 @@ void check_rust_call(const RustCallStatus &status, F error_cb) {
 
 template <typename F, typename EF, typename... Args, typename R = std::invoke_result_t<F, Args..., RustCallStatus *>>
 R rust_call(F f, EF error_cb, Args... args) {
+    ensure_initialized();
```

Comment:
ChatGPT suggestion to ensure the initialize function is only executed once in multi threaded environment:
```
static bool initialized = (ensure_initialized(), true);
```